<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDialog</class>
 <widget class="QDialog" name="NewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewDialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(149, 255, 252);</string>
     </property>
     <property name="locale">
      <locale language="Chinese" country="China"/>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QTreeView" name="treeViewCategory">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>SortBy:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonSmallIcon">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../core.qrc">
         <normaloff>:/core/images/smallicon.png</normaloff>:/core/images/smallicon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonListIcon">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../core.qrc">
         <normaloff>:/core/images/mediumicon.png</normaloff>:/core/images/mediumicon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QListView" name="listViewWizard"/>
   </item>
   <item row="2" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="descriptionText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="descriptionImageLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../core.qrc"/>
 </resources>
 <connections/>
</ui>
